projects/control-service/projects/pipelines_control_service/src/integration-test/java/com/vmware/taurus/graphql/it/GraphQLExecutionsNextRunIT.java

```diff
@@ -0,0 +1,163 @@
+/*
+ * Copyright 2021 VMware, Inc.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.vmware.taurus.graphql.it;
+
+
+import com.vmware.taurus.ServiceApp;
+import com.vmware.taurus.service.JobsRepository;
+import com.vmware.taurus.service.model.DataJob;
+import com.vmware.taurus.service.model.DeploymentStatus;
+import com.vmware.taurus.service.model.JobConfig;
+import org.hamcrest.Matchers;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
+
+import java.time.OffsetDateTime;
+import java.time.ZoneId;
+
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath;
+
+@SpringBootTest(classes = ServiceApp.class)
+@AutoConfigureMockMvc
+public class GraphQLExecutionsNextRunIT {
+
+   @Autowired
+   MockMvc mockMvc;
+
+   @Autowired
+   JobsRepository jobsRepository;
+
+   private final String uri = "/data-jobs/for-team/supercollider/jobs";
```

Comment:
Is the URI hard-coded somewhere in the control service code, or can we change team name?

Reply:
The uri is defined in the open api model definition, we can certainly change the team name, however I'm not sure if we use this uri parameter in the graphQl queries since we retrieve most of the needed query params from the graphql query itself. Maybe we use it to check permissions etc. ?

Reply:
You should extend BaseIT. It already has this propery.

Reply:
You can change the name as well.

Reply:
I don't think I should extend BaseIT here, since this test doesn't need the extra functionality that BaseIT provides (k8s namespace) and in this case extending BaseIT will incur a performance penalty for the test. The only thing that could be reused from BaseIT is this property, however It can't be imported since it's protected in BaseIT and the tests are in different packages.

Reply:
What do you mean by `extra functionality that BaseIT provides (k8s namespace) `?

Reply:
The methods annotated with @BeforeEach and @AfterEach in BaseIT incur a performance penalty, and this test doesn't rely on this setup.

Reply:
Changes as we discussed.